src/Illuminate/Auth/Recaller.php

```diff
@@ -27,7 +27,7 @@ public function __construct($recaller)
     /**
      * Get the user ID from the recaller.
      *
-     * @return string
+     * @return int|string|null
```

Comment:
Please leave this one alone. It's not related (even though the method is called `id`), I think?

Reply:
No problem, change is reverted. I though the changes where need on all three places based on the screenshot.

My background with Laravel core code is quite shallow. So please excuse me for any mistakes :).